client/agent-opentracing/src/main/resources/apmrules/org/jboss/apm-jboss-ee.btm

```diff
@@ -0,0 +1,23 @@
+# Exclude 'onMessage' method as will conflict with instrumentation of JMS/MDB
+
+RULE org.jboss.ee(1) EE Component Start
+CLASS org.jboss.as.ee.component.ProxyInvocationHandler
+METHOD invoke(Object,Method,Object[])
+HELPER org.hawkular.apm.agent.opentracing.OpenTracingManager
+AT ENTRY
+IF !$2.getName().equals("onMessage")
+DO
+  startSpanWithParent(getTracer().buildSpan($2.getName())
+  		.withTag("EJB.uri", truncateAtDelimiter(simpleClassName($1),"$$$")),
```

Comment:
Is there a more reliable way to get the proxied bean name? Also, is this the bean name (interface), or the implementing class name? Would it be interesting to store also the kind of bean being invoked (Home, Local Home, Remote)?